components/bl-audio-recorder/src/helpers.js

```diff
@@ -32,6 +33,81 @@ export const captureMediaDevices = async mediaConstraints => {
   }
 };
 
+export class Timer {
+  constructor(setTime) {
+    this.setTime = setTime;
+    this.currentTime = 0;
+    this.interval = null;
+    this.state = StreamState.INACTIVE;
+    this.startTime = null;
+    this.duration = 0;
```

Comment:
Could be sorted and grouped

```suggestion
    this.setTime = setTime;
    this.state = StreamState.INACTIVE;
    
    this.startTime = null;
    this.currentTime = 0;
    
    this.interval = null;
    this.duration = 0;
```

Or in other way that you think is correct